### PR TITLE
⏪ Revert "Add full_filepath option to fetch_bucket_obj_info"

### PR DESCRIPTION
**Pull request by fiendish**
Reverts d3b-center/d3b-utils-python#7

It turns out I hate this and want to revert it. It explodes the output with redundant information. Can we talk about the motivation for wanting it in the first place?

### CODE REVIEW

File: d3b_utils/aws_bucket_contents.py [hunk 1]
```diff
@@ -59,12 +64,14 @@ def fetch_bucket_obj_info(
             object for object in bucket_contents if object["Size"] > 0
         ]
 
+    if drop_path:
+        for object in bucket_contents:
+            _, _, object["Key"] = object["Key"].rpartition("/")
```
Comment (chris-s-friedman) [inline d3b_utils/aws_bucket_contents.py:69]: I'm not as keen on the idea of modifying `Key. Dropping out the path to the object makes it... no longer the key?

**Comment by chris-s-friedman** [comment]
i don't have a problem with reverting this - all the information for user to build the filepath is already returned.  

iirc, @gsantia 's use case was special where he didn't want to do processing after scraping s3

File: d3b_utils/aws_bucket_contents.py (hunk 1 above)
Comment (chris-s-friedman) [reply d3b_utils/aws_bucket_contents.py:69]: maybe the docstring should note that when you set `drop_path=True`, you lose the ability to pinpoint _where_ in a bucket a file is. User may then run in to an issue where you have two files with the same name. 

And if that is a possibility, I don't think we would want to return that in the function.

**Review by chris-s-friedman — COMMENTED** [review]
Lik i said - I don't have an issue with reverting.... But maybe let it be an option to return the `f"s3://{bucket_name}/{object['Key']}"` (but don't return it by default)

File: d3b_utils/aws_bucket_contents.py (hunk 1 above)
Comment (fiendish) [reply d3b_utils/aws_bucket_contents.py:69]: agree. maybe drop_path should use a different column name

Comment (chris-s-friedman) [reply d3b_utils/aws_bucket_contents.py:69]: we should drop `drop_path`.

**Comment by fiendish** [comment]
> iirc, @gsantia 's use case was special where he didn't want to do processing after scraping s3

yeah but why?

**Comment by gsantia** [comment]
> > iirc, @gsantia 's use case was special where he didn't want to do processing after scraping s3
> 
> yeah but why?

I thought it'd be nice to avoid iterating over the same data more than once 🤴

**Review by gsantia — APPROVED** [review]
all good 👍

**Pull request merged**